aequilibrae/paths/bpr.pyx

```diff
@@ -1,27 +1,31 @@
 from libc.math cimport pow
 from cython.parallel import prange
 
-def bpr(congested_times, link_flows, capacity, fftime, alpha, beta):
+def bpr(congested_times, link_flows, capacity, fftime, alpha, beta, cores):
+    cdef int c
```

Comment:
Is there a reason you declare and initialise in different places?

Reply:
Yes.  My poor programming skills. Solved now (and incorporated elsewhere)